packages/wp-now/src/run-cli.ts

```diff
@@ -87,22 +90,25 @@ export async function runCli() {
 			}
 		)
 		.command(
-			'php <file>',
-			'Run the php command passing the arguments for php cli',
+			'php [..args]',
+			'Run the php command passing the arguments to php cli',
 			(yargs) => {
 				commonParameters(yargs);
-				yargs.positional('file', {
-					describe: 'Path to the PHP file to run',
-					type: 'string',
-				});
+				yargs.strict(false);
```

Comment:
Huh! I tried this yesterday but it disabled strict mode for `wp-now start` too. It doesn't seem to be the case on your PR, though:

```
$ nx preview wp-now start --foo=bar
wp-now start
Start the server
Options:
      --version  Show version number                                   [boolean]
      --path     Path to the PHP or WordPress project. Defaults to the current w
                 orking directory.                                      [string]
      --php      PHP version to use.                                    [string]
      --wp       WordPress version to use: e.g. '--wp=6.2'              [string]
      --port     Server port                                            [number]
  -h, --help     Show help                                             [boolean]
Unknown argument: foo
```

Reply:
The reason is that I'm setting `strict` to `false` inside the command declaration `.command( 'php [..args]',`, so it's limited to this scope.